packages/apps/examples/icy-graphql-client/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["../../../../.eslintrc.json"],
```

Comment:
huh I actually just learnt that specifically for eslint config, trying to override turn off the `@typescript-eslint/ban-ts-comment` rule that because of NX's base override of eslint, you need to override per project these rules or do more extensive eslint setup 🤦 
https://github.com/nrwl/nx/issues/5866